data/json/items/generic/bathroom_house.json

```diff
@@ -317,7 +317,7 @@
       { "id": "bbottle_5", "text": "A colorful bottle of body and hair wash marketed for children." },
       { "id": "bbottle_6", "text": "A plain white bottle made to contain the cheapest shampoo." },
```

Comment:
I guess this could be removed as it's just a copy of `bbottle_2`.

Reply:
Removed and renamed the last two ids. If those should stay the existing ids, I can change them back.